pages/dao/[symbol]/proposal/components/instructions/InitializeController.tsx

```diff
@@ -64,17 +62,19 @@ const InitializeController = ({
       isValid &&
       programId &&
       form.governedAccount?.governance?.info &&
+      form.mintDecimals &&
       wallet?.publicKey
     ) {
-      const createIx = createInitializeControllerInstruction(
+      const initializeControllerIx = createInitializeControllerInstruction(
         form.governedAccount?.governance.info.governedAccount,
-        form.mintSymbol || '',
-        form.mintDecimals || 9,
+        form.mintDecimals,
```

Comment:
No default values, on top of it it errored later. This are important parameters we need to be able to rely on the UI.

I've removed the Symbol that was not used. And the Decimal value wasn't used but always used a default value of 6. Danger